lib/package.js

```diff
@@ -394,8 +394,10 @@ const ar = require('ar-async'),
 
             function _removeTmpDir(next){
                 log.info("package#analyzeIPK()#_removeTmpDir()");
-                fs.rmdirSync(tmpDirPath, {recursive: true, force:true});
-                next();
+                rimraf(tmpDirPath, function(err) {
+                    log.verbose("package#analyzeIPK()#_removeTmpDir()", "removed " + tmpDirPath);
+                    next(err, null);
```

Comment:
Could you use "next(err)" instead of next(err, null);

Reply:
I've updated this.